### DISCUSSION TITLE
package not installing

### DISCUSSION THREAD

**Discussion by kunalsingh1301** [visgl/react-google-maps#152] [General]
```
npm i @vis.gl/react-google-maps 

At line:1 char:7
+ npm i @vis.gl/react-google-maps
+       ~~~~
The splatting operator '@' cannot be used to reference variables in an expression. '@vis' can be used only as an argument  
to a command. To reference variables in an expression use '$vis'.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : SplattingNotPermitted
```

**Comment by kunalsingh1301** [comment]
install using cmd

**Comment by usefulthink** [comment]
This looks like a PowerShell specific problem, and you probably need to quote the package name since PowerShell has a special meaning for the `@`.

I think this would be something PowerShell users are aware of - do you think this should be mentioned specifically in the readme? (I'm not on windows and had little contact with PowerShell so far)

**Reply by kunalsingh1301** [reply]
Yes it would be great if mentioned in readme without proper search text it's little difficult to find

**Comment by usefulthink** [comment]
Done: https://github.com/visgl/react-google-maps/pull/168

**Comment by GibranKhalil** [comment]
try: npm i "@vis.gl/react-google-maps"
